README.md

```diff
@@ -27,6 +27,8 @@ If you see anything that is no longer maintained or is not a good fit, please su
 ### C
 
 - [ChoTot](https://www.chotot.com/) - [github](https://github.com/ChoTotOSS)
+- [CyberAgent, Inc.](https://www.cyberagent.co.jp/en/) - [Hanoi Development Center](https://www.facebook.com/ca.devcenter.vietnam) - Use Go to build a wide range of products with multi-million user bases.
```

Comment:
please remove facebook url; and only use website and github

Reply:
fixed https://github.com/golang-vietnam/companies/pull/54/commits/0aec97dae1edab3378742bfb977c2003f99dc6a2